lambdas/data-migration2/src/storeErrors.ts

```diff
@@ -0,0 +1,27 @@
+import { s3PutObject } from '@cumulus/aws-client/S3';
+
+/**
+ * Store migration errors JSON file on S3.
+ *
+ * @param {string} bucket
+ * @param {string[]} message
+ * @param {string} recordClassification
+ * @param {string | undefined} stackName
+ * @returns {void}
+ */
+export const storeErrors = async (
+  bucket: string,
+  message: string[],
+  recordClassification: string,
+  stackName?: string) => {
+  const file = `{
+    "errors": ${JSON.stringify(message)}
+  }`;
+  const filename = `data-migration2-${recordClassification}-errors.json`;
+  const key = `${stackName}/${filename}` || filename;
```

Comment:
What is the logic here doing? Effectively, even if `stackName` was undefined, you'd be doing:

```
`/${filename}` || filename
```

Whether filename has a value or not, nothing after the `||` is ever reached, so I think it's unnnecessary